index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PortFolio Project</title>
+
+  <!-- Linking the page to the style file A.css -->
+  <link rel="stylesheet" href="styles/style.css">
+  
+</head>
+<body>
+  <header class="toolbar">
+    <section class="logo">
+      MZK
+    </section>
```

Comment:
- You have done a good job so far,  but to help improve the semantics of your site, I suggest using the link tag, `a` tag instead of the `section` tag. Please fix this 😄

Reply:
Ok done